packages/crayons-core/src/components/data-table/data-table.tsx

```diff
@@ -994,11 +994,14 @@ export class DataTable {
    */
   renderPredefinedVariant(columnVariant: string, cellValue: any) {
     let template: JSX.Element;
-    if (PREDEFINED_VARIANTS_META[columnVariant]) {
-      template = this.renderWebComponent(
-        PREDEFINED_VARIANTS_META[columnVariant].componentName,
-        cellValue
-      );
+    if (columnVariant === 'anchor') {
+      template = <fw-custom-cell-anchor {...cellValue}></fw-custom-cell-anchor>;
```

Comment:
I see that `slotText` has been used to pass a slot to the custom variants in the implmentation of `renderWebComponent` . Is this being handled here ?

Can you also pls remove `renderWebComponent` method if this is no longer in use.

Reply:
Yes, it is handled in case of custom variants. 

renderWebComponents is a function we might definitely be needing later. I would recommend that we retain this function for now.

Reply:
Thanks Vignesh